build.sbt

```diff
@@ -180,7 +181,8 @@ lazy val ratatoolSampling = project
       "com.twitter" %% "algebird-core" % algebirdVersion,
       "joda-time" % "joda-time" % jodaTimeVersion,
       "org.scalacheck" %% "scalacheck" % scalaCheckVersion,
-      "org.scalatest" %% "scalatest" % scalaTestVersion % "test"
+      "org.scalatest" %% "scalatest" % scalaTestVersion % "test",
+      "com.google.flogger" % "flogger-system-backend" % floggerVersion % "test"
```

Comment:
If the issues occurs in ParquetIOTest can we move this dep to ratatool-extras

Reply:
The [docs](https://github.com/google/flogger#1-add-the-dependencies-on-flogger) for flogger say that "the dependency on flogger-system-backend is only required to be included when the binary is run", so I think there is a transitive dependency that is using flogger, but is excluding flogger-backend. Since we aren't supplying a logging backend, flogger is defaulting to throwing an exception.

The strange thing is that you weren't able to replicate it and the tests pass on CI.